code_to_gast/helpers/js_helpers.py

```diff
@@ -73,11 +73,21 @@ def bool_op_to_gast(node):
 Used for functions called on objects and std funcs like console.log
 """
 def member_expression_to_gast(node):
-  if node.property.name == "log":
+  if node.object.name == "console" and node.property.name == "log":
     return {"type": "logStatement"}
-
-  gast = {"type": "attribute", "id": node.property.name}
-  gast["value"] = js_router.node_to_gast(node.object)
+  
+  gast = {"value": js_router.node_to_gast(node.object)}
+  func_name = node.property.name
```

Comment:
Adding a comment here explaining that this will mistranslate function calls that aren't Array.prototype.pop etc... might be useful down the line. Also consider adding a TODO with a link to a taiga ticket.

Reply:
Updated